src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -144,6 +153,39 @@ void courthousesWithRegionAndSecurityGroupsGet() throws Exception {
 
     }
 
+    @Test
+    @Transactional
+    void courthousesGetWithRegionId() throws Exception {
```

Comment:
How does this test differ in purpose from the existing test `courthousesGetAll`?

Reply:
courthouseGetAll assertsTrue by checking a court exists in a list and providing the content body only. It sets up the created and last modified date and time but does not compare the actual values that it sets.
courthouseGet sets up regions for courthouses and validates if the region_id exists and the value. This can be further extended if required.